ui/desktop/electron-app/src/services/cache-daemon-manager.js

```diff
@@ -117,9 +117,18 @@ class CacheDaemonManager {
     return generateErrorPromise(stderr);
   }
 
-  search(requestData) {
+  async search(requestData) {
+    const start = Date.now();
+    // Log request data, but clean up token from log
+    const { auth_token_id, token, ...logRequestData } = requestData;
+
     if (isWindows()) {
-      return searchCliCommand(requestData);
+      const result = await searchCliCommand(requestData);
+
+      const end = Date.now();
+      log.debug(`Search request took ${end - start} ms`, logRequestData);
```

Comment:
do we need to mention that this is from `windows` or will that info already be there?

Reply:
What do you mean by this is from `windows`?

Reply:
for us to identify that this comes from windows env, since we have two sets of logs one for windows and other OS

Reply:
I'm not sure I'm following, there'll only ever be one set of logs. On windows this debug statement will run and on other OS's the other debug statement will run. They'll never both run

Reply:
you are right, I was thinking of a situation where we get a dump of logs from different machines to debug but that will never be the case. sorry for the confusion.